server/app/serializers/host_node_serializer.rb

```diff
@@ -0,0 +1,83 @@
+class HostNodeSerializer < KontenaJsonSerializer
+
+  attribute :id
+  attribute :connected
+  attribute :created_at
+  attribute :updated_at
+  attribute :last_seen_at
+  attribute :name
+  attribute :os
+  attribute :engine_root_dir
+  attribute :driver
+  attribute :kernel_version
+  attribute :labels
+  attribute :mem_total
+  attribute :mem_limit
+  attribute :cpus
+  attribute :public_ip
+  attribute :private_ip
+  attribute :agent_version
+  attribute :docker_version
+  attribute :peer_ips
+  attribute :node_number
+  attribute :initial_member
+  attribute :grid
+  attribute :resource_usage
+
+  def id
+    object.node_id
+  end
+
+  def last_seen_at
+    object.last_seen_at.try(:iso8601)
+  end
+
+  def engine_root_dir
+    object.docker_root_dir
+  end
+
+  def peer_ips
+    if object.grid
+      object.grid.host_nodes.ne(id: object.id).map{|n|
```

Comment:
This logic is now duplicated here and in the `server/app/views/v1/host_nodes/_host_node.json.jbuilder`... refactor to a `HostNode` model method?